app/src/ui/app.tsx

```diff
@@ -1412,6 +1413,24 @@ export class App extends React.Component<IAppProps, IAppState> {
     this.props.dispatcher.selectRepository(repository)
     this.props.dispatcher.closeFoldout(FoldoutType.Repository)
   }
+
+  private onViewCommitOnGitHub = async (SHA: string) => {
+    const repository = this.getRepository()
+
+    if (
+      !repository ||
+      repository instanceof CloningRepository ||
+      !repository.gitHubRepository
+    ) {
+      return
+    }
+
+    const baseURL = repository.gitHubRepository.htmlURL
+
+    if (baseURL) {
```

Comment:
It shouldn't, but since `htmlURL` can be string or null, I have to check for it.